lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java

```diff
@@ -843,7 +843,11 @@ public SimplePage getPage(long pageId) {
 	}
 
 	public List<SimplePage> getSitePages(String siteId) {
-	    DetachedCriteria d = DetachedCriteria.forClass(SimplePage.class).add(Restrictions.eq("siteId", siteId)).add(Restrictions.isNull("owner"));
+	    DetachedCriteria d = DetachedCriteria.forClass(SimplePage.class).add(Restrictions.eq("siteId", siteId))
+			        .add(Restrictions.disjunction()
+						.add(Restrictions.isNull("owner"))
+					    .add(Restrictions.conjunction().add(Restrictions.isNotNull("owner")).add(Restrictions.eq("owned", true)))
```

Comment:
Can't this just be: `.add(Restrictions.eq("owned", true))` as you test for owner being `null` on the previous option.